pkgs/development/python-modules/s3-credentials/default.nix

```diff
@@ -21,13 +22,14 @@ buildPythonPackage rec {
     owner = "simonw";
     repo = pname;
     rev = "refs/tags/${version}";
-    hash = "sha256-kzu+ySeAi+nRIJSBSJUQzNHD7KVANlWoi0OWKXkc0/8=";
+    hash = "sha256-vKOcOSt9vscj5ixrHZGL6PRun/x38JLbni75nw2YAbg=";
   };
 
   propagatedBuildInputs = [
     click
     click-default-group
     boto3
+    moto
```

Comment:
This goes into checkInputs

Reply:
Ah, thanks for catching that. Updated :)